ATTRIBUTION.md

```diff
@@ -590,6 +590,38 @@ The data are used to fit model for temperature-dependent fossil fuel usage.
 ##### General Purpose
 The data are used to fit model for temperature-dependent fossil fuel usage.
 
+##### Source
+* Name: Fraction Target for Reisidential and Commercial heat pump possession as main heating device
+* Author: EIA
+* Description: Fraction of residential or commercial floor space using heat pump as main heating device for Census Regions
+* Source: https://www.eia.gov
+* Exact source location:
+  * RECS2015: https://www.eia.gov/consumption/residential/data/2015/
+  * RECS2015: https://www.eia.gov/consumption/residential/data/2020/
+  * CBECS2012: https://www.eia.gov/consumption/commercial/data/2012/
+  * CBECS2018: https://www.eia.gov/consumption/commercial/data/2018/
+
+##### Destination
+* Modifications to source file(s): Interpolate 2019 values based on RECS2015 and RECS2020, and extrapolate 2019 values based on CBECS2012 and CBECS 2018.
+* Location: ***prereise/gather/demanddata/bldg_electrification/data/frac_target_hp_res.csv***
+            ***prereise/gather/demanddata/bldg_electrification/data/frac_target_hp_com.csv***
+
+##### General Purpose
+The data are used to fit model for current heat pump penetration rate.
+
+##### Source
+* Name: Natural Gas Prices
+* Author: EIA
+* Description: Annual average natural gas price (residential) of states
+* Source: https://www.eia.gov
+* Exact source location: 
+  * https://www.eia.gov/dnav/ng/ng_pri_sum_dcu_nus_a.htm
+##### Destination
+* Location: ***prereise/gather/demanddata/bldg_electrification/data/state_ng_residential_price_2019.csv***
+
+##### General Purpose
+The data are used in the linear regression model for estimating current heat pump penetration rate. 
+
```

Comment:
How about the AHS survey data `hp_ahs.xlsx`?

Reply:
it's listed below, under U.S. Census Bureau